client/src/app/pages/Artifacts/components/ArtifactCard.tsx

```diff
@@ -0,0 +1,82 @@
+import type { ArtifactVerificationViewModel } from "@app/queries/artifacts.view-model";
+import {
+  Card,
+  CardHeader,
+  Content,
+  Flex,
+  FlexItem,
+  Button,
+  Label,
+  CardBody,
+  Tabs,
+  Tab,
+  TabTitleText,
+} from "@patternfly/react-core";
+import { ArtifactAttestations } from "./ArtifactAttestations";
+import { ArtifactSignatures } from "./ArtifactSignatures";
+import { ArtifactSummary } from "./ArtifactSummary";
+import { verificationStatusToLabelColor } from "@app/utils/utils";
+import { useState } from "react";
+import type { ImageMetadataResponse } from "@app/client";
+import { ExternalLinkSquareAltIcon } from "@patternfly/react-icons";
+
+interface IArtifactCard {
+  artifact: ImageMetadataResponse;
+  verification: ArtifactVerificationViewModel;
+}
+
+export const ArtifactCard = ({ artifact, verification }: IArtifactCard) => {
+  const [activeTabKey, setActiveTabKey] = useState<string | number>(0);
+  const { overallStatus } = verification.summary;
+  const { label: verificationLabel, color: verificationLabelColor } = verificationStatusToLabelColor(overallStatus);
+
+  const handleTabClick = (
+    _event: React.MouseEvent<unknown> | React.KeyboardEvent | MouseEvent,
+    tabIndex: string | number
+  ) => {
+    setActiveTabKey(tabIndex);
+  };
+
+  return (
+    <Card style={{ overflowY: "hidden" }}>
+      <CardHeader>
+        <Content style={{ overflow: "hidden", textOverflow: "ellipsis" }}>
+          <Flex className="border">
+            <FlexItem>
+              Artifact:{" "}
+              <Button
+                variant="link"
+                href={`https://${artifact.image}`}
```

Comment:
This is a best guess for now, as this value will be provided by the backend in a separate PR, and then we can update this button when it's ready.